tests/ui_tests/test_ui_add_component_window.py

```diff
@@ -65,6 +65,20 @@
     else:
         NO_PIXEL_OPTIONS[component_class] = i
 
+# Select a subset of the component class to use in parameterised tests
+# Should include any for which the UI is specialised
+_components_subset = {"NXdetector", "NXdisk_chopper", "NXsensor"}
```

Comment:
list literal instead of dict? both work, but lists are more probably more pythonic

Reply:
It's a set, not a dict, seems most appropriate here to me.